tempto-core/src/test/groovy/com/teradata/tempto/assertions/QueryAssertTest.groovy

```diff
@@ -56,6 +57,11 @@ public class QueryAssertTest
     }
   }
 
+  def setupSpec()
+  {
+    System.setProperty(TEST_CONFIGURATION_URI_KEY, "/configuration/global-configuration-tempto.yaml");
```

Comment:
It is not just simple. QueryAssert is using static methods it is hacky to use mocking there.